content/_en/policy/rules-of-use.md

```diff
@@ -0,0 +1,157 @@
+- How we use your information and your rights to that information, and
+- The conditions you agree to when you take certain actions on the login.gov service.
+
+# 1. General Service Definition
+
+The login.gov service offers the public secure and private online access to participating government programs. With one login.gov account, you can sign into services offered by multiple government agencies. Our goal is to make managing federal benefits, services, and applications easier and more secure.
+
+The login.gov service partners with other federal agencies ("partners") to allow users to access those services with just one login.gov account, eliminating the need to create many separate accounts across government.
+
+The login.gov service protects your account by implementing strong security measures and protects your privacy by collecting the minimally necessary information from you and, in turn, revealing to partners only the information necessary for those partners to execute their service. And login.gov never shares anything about you with a partner without your explicit consent and you can revoke that consent at any time.
+
+Some government applications that use login.gov require users to verify their identities. This means that you must prove that you are who you say you are. That proof helps ensure that only the right people get access to sensitive information.
+
+You will only be asked to verify your identity the first time you sign into certain government applications. You only need to verify your identity once for your login.gov account. After you verify your identity with login.gov for one government application, you don't need to do it again for other government applications. By providing the login.gov service with the information needed to verify your account, you authorize it to use that information to conduct activities necessary to ensure your identity, including sharing information with certain third parties, namely identity proofing services. The login.gov service requires those parties to hold your information confidentially and not to use it for other purposes, as detailed in our [privacy policy]({% link _en/policy/privacy.md %}).
+
+The login.gov service employs recognized security and privacy best practices.
+
+Prior to using the login.gov service, you are required to agree to these terms of service. When the login.gov service changes its terms of service, you will be given the option to agree or to decline the updated terms of service. Similarly, when conducting certain activities in the login.gov service, such as providing personal information for the first time, the login.gov service may require you to re-confirm your understanding of how it uses your information.
+
+# 2. Your Agreement
+
+In addition to details about your consent stated elsewhere in these terms, by accepting these terms and using the login.gov service, you agree that:
+
+1. Are not a child under 13 years of age,
```

Comment:
I see it's written this way in the document, but the grammar here seems incorrect.

```suggestion
1. You are not a child under 13 years of age,
```

Not sure who'd need to sign off on such a change though.